services/galley/src/Galley/Data/LegalHold.hs

```diff
@@ -73,13 +73,14 @@ selectPendingPrekeys uid =
 dropPendingPrekeys :: MonadClient m => UserId -> m ()
 dropPendingPrekeys uid = retry x5 (write Q.dropPendingPrekeys (params Quorum (Identity uid)))
 
-getUserLegalHoldStatus :: MonadClient m => UserId -> m UserLegalHoldStatus
-getUserLegalHoldStatus uid = do
-    result <- retry x1 (query1 Q.selectUserLegalHoldStatus (params Quorum (Identity uid)))
+getUserLegalHoldStatus :: MonadClient m => TeamId -> UserId -> m UserLegalHoldStatus
+getUserLegalHoldStatus tid uid = do
```

Comment:
If you like, you can drop this function and call `getMember` (or whatever it's called) instead.  Less code, more good!  :-)

(I haven't checked if there is such a more general function, though.)